app/assets/stylesheets/components/_email-signup-section.scss

```diff
@@ -0,0 +1,38 @@
+.app-c-email-signup {
+  @include govuk-responsive-margin(8, "top");
+  padding: govuk-spacing(6);
+  background-color: govuk-colour("light-grey", $legacy: "grey-4");
+  border: 1px solid $govuk-border-colour;
+}
+
+.app-c-email-signup--full-width {
+  margin-top: 0;
+  border-left: 0;
+  border-right: 0;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.app-c-email-signup--transparent_background {
```

Comment:
Writing this comment mainly to remind myself to check this - when it's transparent, do we still need the left/right padding?